### PR TITLE
The city_stops model was created

### PR DESCRIPTION
**Pull request by RichMicke**
The city_stops model was created, it still needs to be finished and added values

### CODE REVIEW

File: lib/trajectories/route_22.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (mtopete1996) [inline lib/trajectories/route_22.json]: This what?? Esto qué?

**Review by mtopete1996 — APPROVED** [review]
You have an extra file 🤔

File: test/fixtures/city_stops.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (jesusMava) [inline test/fixtures/city_stops.yml]: are we going to use fixtures?

**Review by jesusMava — APPROVED** [review]

**Review by mtopete1996 — CHANGES_REQUESTED** [review]
You still have files that don't belong here

**Review by mtopete1996 — APPROVED** [review]

**Pull request merged**